pkgs/development/tools/rust/sqlx-cli/default.nix

```diff
@@ -1,17 +1,17 @@
-{ stdenv, lib, rustPlatform, fetchFromGitHub, pkg-config, openssl, SystemConfiguration, CoreFoundation, Security, libiconv }:
+{ stdenv, lib, rustPlatform, fetchFromGitHub, pkg-config, openssl, SystemConfiguration, CoreFoundation, Security, libiconv, testVersion, sqlx-cli }:
```

Comment:
```suggestion
{ stdenv, lib, rustPlatform, fetchFromGitHub, pkg-config, openssl, SystemConfiguration, CoreFoundation, Security, libiconv, testers, sqlx-cli }:
```

Reply:
Replaced as of 61e1418f8affb9616f71d26c14e01824a722bcde - is there any context you can share for when I should prefer this style of call vs a direct import like I had in place originally?

Reply:
Though ofborg does not really catch everything, we try to avoid using aliases:

https://github.com/NixOS/nixpkgs/blob/41808d42d21f685e3fd3cdb2c9f04082f0818e30/pkgs/top-level/aliases.nix#L1256